R/valid_dlw_load.R

```diff
@@ -0,0 +1,65 @@
+valid_dlw_load <- function(inv, path) {
+
+  #~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  # computations   ---------
+  #~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  # Order alphabetically
+  file_dta <- basename(inv$fullname)
+  file_qs <- sub("\\.dta$", ".qs", file_dta)
+  file_qs <- sort(file_qs)
+
+  # Load survey files
+  ls_svy     <- lapply(1:n, \(x) qs::qread(file.path(path, file_qs[x])))
+
+  poss_data_to_df <- purrr::possibly(.f = data_to_dt,
+                                     otherwise = NULL)
+
```

Comment:
Please, make user of the `logging` system of `pipfun`. It was design with the objective that all the members in our team use it in their codes.